package.json

```
@@ -30,7 +34,7 @@
     "@babel/plugin-transform-modules-commonjs": "7.19.6",
     "@babel/preset-env": "7.19.4",
     "@iktakahiro/markdown-it-katex": "4.0.1",
-    "@types/node": "16.x",
+    "@types/node": "~16.18.3",
```

```suggestion
        "@types/node": "14.x || 16.x || 18.x",
```

I am not 100% sure, but i think this is how we are supposed to specify the version range for this if we are targeting all LTS lines. The version specified in the `.nvmrc` file in the repo root should be what is used for the version of this dependency as seen in the version selected for the lockfile. We need to setup CI to test all those versions too.